baggage/HTTP_HEADER_FORMAT.md

```diff
@@ -101,3 +102,17 @@ For example, if you have non-production requests that flow through the same serv
 ```
 baggage: isProduction=false
 ```
+# Mutating baggage
+A system receiving a `baggage` request header SHOULD send it to outgoing requests.
+A system MAY mutate the value of this header before passing it on.
+
+As the producer and consumer of `baggage` SHOULD be within the same logical system and the content of the
+key/value pairs is opaque to the spec, additional proprietary rules for mutations MAY be defined.
+Example: Defining a precedence when deduplicating keys.
+
+The following mutations are allowed:
+
+* **Add a new key/value pair.** A key/value pair MAY be added.
+* **Update an existing value.** The value for any given key MAY be updated.
+* **Delete a key/value pair.** Any key/value pair MAY be deleted.
+* **Deduplicating the list.** Duplicate keys MAY be removed.
```

Comment:
This is implied by delete is it not?

Reply:
I would suggest extendind the dedup item with the "SHOULD" use last value when possible

Reply:
I will change the wording on dedup to [..] whereas the last entry SHOULD have precedence